scripts/create_release.py

```
@@ -0,0 +1,99 @@
+    try:
+        current_version = get_current_version()
+        print(f"Current version: {current_version}")
+        print(f"New version: {args.version}")
+        
+        if args.update_version:
+            update_version(args.version)
+        
+        if args.create_notes:
+            create_release_notes(args.version, args.type)
+            
+        print(f"\n📋 Next steps:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)